src/suvela/PracticeArray.java

```diff
@@ -0,0 +1,60 @@
+package suvela;
+
+public class PracticeArray {
+	int min(int[] array) {
+		int index=0;
+		int min = array[index];
+		for (index = 0; index < array.length; index++) {
+			if (array[index] < min)
+				min = array[index];
+		}
+		return min;
+
+	}
+
+	int max(int[] array) {
+		int index=0;
+		int max = array[index];
+		for (index = 0; index < array.length; index++) {
+			if (array[index] > max)
+				max = array[index];
+		}
+		return max;
+	}
+
+	void display(int[] numbers) {
+		for (int index = 0; index < numbers.length; index++) {
+			System.out.println(numbers[index]);
+		}
+	}
+
+	void reverse(int[] numbers) {
+		for (int index = numbers.length - 1; index >= 0; index--) {
+			System.out.println(numbers[index]);
+		}
+	}
+
+	int average(int[] array) {
+		int sum = 0;
+		for (int index = 0; index < array.length; index++) {
+			sum = sum + array[index];
+		}
+		return sum / array.length;
+	}
+
```

Comment:
looks like you missed one scenarios
g. Method to find average of min and max number from array.